src/pages/commits/InactiveCommits/InactiveCommits.test.tsx

```diff
@@ -0,0 +1,122 @@
+import userEvent from "@testing-library/user-event";
+import { render, act } from "test_utils/test-utils";
+
+import { InactiveCommits } from ".";
+
+const RenderInactiveCommits = (versions) => (
```

Comment:
this is a React component that accepts the same props as InactiveCommits. I think we can replace RenderInactiveCommits usage with <InactiveCommits />